Gaijinizer patch/Map071.txt

```diff
@@ -123,39 +129,42 @@ processing. It is a little more on the surface
  果てには非人道的な人体実験に手を出してしまい、
  政府から危険人物指定を受け、一族からも追い出された。
 > CONTEXT: Map071/events/1/pages/0/70/Dialogue/yuuma
-However, excessive biological experiments,
-possession of dangerous drugs.
-In the end, he handed out an inhuman human
-experiment, The government designated him as a
-dangerous person and was expelled from his
-family.#MTLed
+However, there were biological experiments that
+went too far, and possession of dangerous drugs.
+In the end, she got involved in inhumane human
+experimentation, and the government designated her
+as a dangerous individual, and she was expelled
+from her clan.#Ron
```

Comment:
Flow:
>_**However, her biological experiments went too
far, and she was in possession of dangerous and highly
illegal drugs. In the end, she got involved in unethical
human experimentation, which caused the government
to designate her as a massive security risk. She was
expelled from her clan shortly afterwards.**_

("shortly afterwards" is a minor liberty not in the original text, but it feels appropriate to say here.)